content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -79,55 +79,54 @@ Recent key learnings:
 
 ## Strategy and Plans
 
-### FY22 Q4 goals
-
-These tie back to [FY22 Q4 Product/Engineering OKRs](../../../goals/2022_Q4.md#product-engineering)
+### Themes
 
-**Objective: Make the power of our features easier to find and use**
+**Unlock dependency navigation**
 
-- **KR:** Measure and increase WAU for customer facing product teams by 15%. For most teams this will require understanding and improving time-to-value as a driver for increasing active users.
+We believe this is the global code graph’s killer feature. Most developers think in terms of package and dependency versions rather than focusing on external repositories. Supporting precise dependency navigation elevates the code navigation experience to a new level of cross-project analysis. The ability to navigate to any third party dependency a repository references is key to use cases like [Developer onboarding](../use-cases/dev-onboarding.md) where time to value is directly related to how fast a developer is able to navigate and undestand a new codebase (including the dependencies it relies on).
 
-**Objective: Level up our enterprise-ready features**
+Taking it a step further, adding support for a wider set of artifactory hosts opens the door to a whole new array of cross-team features that support our use cases. These include (but are not limited to):
 
-- **KR:** Enable out-of-the-box cross-repo and dependency navigation for 3 customers by successfully moving them to on-prem auto-indexing.
-- **KR:** Improve precise code intelligence actions coverage by going from 3 to 5 languages (adding Kotlin and TS/JS), representing the languages in ~50% of all actions.
+- Accelerating [Developer onboarding](../use-cases/dev-onboarding.md) by enabling search for dependency versions and across transitive dependencies.
+- Monitoring [Code Health](../use-cases/code-health.md) by creating dependency tracking insights to detect deprecated dependency usage.
+- [Fixing security vulnerabilities](../use-cases/fixing-security-vulnerabilities.md) and helping [Incident response](../use-cases/incident-response.md) by searching and finding references to vulnerable dependency versions or errors, later applying a batch fix using Batch Changes.
```

Comment:
I think the first point is good, having precise or search intel over *all* your code helps both those uses cases, precise being able to help on a much deeper level due to the ability to see the full transitive dependency graph. 

The latter "later applying a batch fix" is likely going to rely on precise intel no?  So perhaps we change it something along the lines of "being able to have an available fully semantic graph of the code in question would allow other products such as BC to do X"